apps/backend/src/mcp/tools/container_management.py

```
@@ -172,71 +166,73 @@ async def list_containers(
                         status_str = f"Up ({health})"
                     else:
                         status_str = "Up"
-                        
+
                 # Extract creation and start times for uptime calculation
                 created = container_data.get("Created", "")
```

Ruff (0.12.2)

Local variable `created` is assigned to but never used

Remove assignment to unused variable `created`

(F841)